package.json

```diff
@@ -13,10 +13,20 @@
     "chai": "^4.2.0"
   },
   "devDependencies": {
-    "cypress": "^4.5.0"
+    "cypress": "^4.5.0",
+    "cypress-multi-reporters": "^1.4.0",
+    "mocha": "^8.0.1",
+    "mochawesome": "^6.1.1",
+    "mochawesome-merge": "^4.1.0",
+    "mochawesome-report-generator": "^5.1.0"
   },
     "scripts": {
-      "chrome:headless": "cypress run --headless --browser chrome",
-      "chrome:head": "cypress run --browser chrome"
+      "clean:reports": "rm -R -f cypress/reports && mkdir cypress/reports",
+      "pretest": "npm run clean:reports",
+      "scripts": "cypress run",
+      "combine-reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+      "generate-report": "marge cypress/reports/report.json -f report -o cypress/reports",
+      "posttest": "npm run combine-reports && npm run generate-report",
```

Comment:
Lines 27, 28 and 29:
Combine them in 1 command, e.g.:
`"cypress:report": "combineReports && generateReports"`

Reply:
Take a look at   "posttest", please. 
This is the same, isn't it?

  `"posttest": "npm run combine-reports && npm run generate-report"`

Reply:
could we add also `open cypress/reports/report.html`?

Reply:
You can add, but it will work only for PASSED builds. If any of tests will fail during `cypress run` command, then the build will be FAILED and the 3-rd command will not run => reports will not be generated.
So it will be better to have 2 commands: 1 - for test run, 2- for reporting.